add cheatsheet for tidyverse

Huimin-alice opened the pull request:
add a contributing file and a picture.

Commit:
add cheatsheet for tidyverse

RandomWalker300 commented on tidyverse_contribution.Rmd:
```diff
@@ -0,0 +1,6 @@
+#cheatsheet for tidyverse
```
1. line 1: add a space after hashtag
2. line6: change your .png path to "resources/tidyverse_tutorial/tidyverseFinal.png"
3. remove .DS_Store, and resources/.DS_Store

Huimin-alice replied:
all problems listed are fixed, please check.

RandomWalker300 reviewed (changes requested):
line 1: please add a space after hashtag
line6: please change your .png path to "resources/tidyverse_tutorial/tidyverseFinal.png"
please remove .DS_Store, and resources/.DS_Store

Commit:
line 1: add a space after hashtag
line6: change your .png path to "resources/tidyverse_tutorial/tidyverseFinal.png"

Huimin-alice committed:
Delete .DS_Store

Huimin-alice committed:
Delete .DS_Store

Huimin-alice commented:
all the problem listed as fixed, please check.

RandomWalker300 committed:
Update _bookdown.yml

add tidyverse_cheatsheet to _bookdown.yml

RandomWalker300 committed:
Update _bookdown.yml

RandomWalker300 committed:
Updata _bookdown.yml

skyetim committed:
Merge branch 'master' into tidyverse_cheatsheet

skyetim commented:
🎉 Congrats! Your contribution has been merged!
